src/stats/manager_impl.go

```diff
@@ -0,0 +1,119 @@
+package stats
+
+import (
+	pb_struct "github.com/envoyproxy/go-control-plane/envoy/extensions/common/ratelimit/v3"
+	"github.com/envoyproxy/ratelimit/src/settings"
+	gostats "github.com/lyft/gostats"
+	logger "github.com/sirupsen/logrus"
+)
+
+func NewStatManager(store gostats.Store, s settings.Settings) *ManagerImpl {
+	serviceScope := store.ScopeWithTags("ratelimit", s.ExtraTags).Scope("service")
+	return &ManagerImpl{
+		store:                store,
+		rlStatsScope:         serviceScope.Scope("rate_limit"),
+		legacyStatsScope:     serviceScope.Scope("call.should_rate_limit_legacy"),
+		serviceStatsScope:    serviceScope,
+		detailedMetricsScope: serviceScope.Scope("rate_limit").Scope("detailed"),
```

Comment:
should this scope be initialised only if `detailed` is set to true?

Reply:
Removed all mentions of `detailedMetrics` they will be added on the DetailedMetrics PR